client/scripts/app.ts

```diff
@@ -417,6 +440,39 @@ export function initCommunity(communityId: string): Promise<boolean> {
   }
 }
 
+export async function initTemporaryTokenChain(address: string): Promise<boolean> {
+  // todo token list in localstorage
+  let tokenLists = await app.tokens.getTokensFromLists()
+  let token = tokenLists.find(o=>{ return o.address === address })
+
+  if(!token) { return false }
+
+  app.threads.initialize([],0,0,true)
+
+  return createTemporaryTokenChain(
+    new NodeInfo(
```

Comment:
May want to clean up this init by adding some sort of static method on the model that takes a token and emits what you need.